.travis.yml

```diff
@@ -17,7 +17,7 @@ notifications:
 install:
 - mvn -version -B
 script:
-- mvn -s .travis.maven.settings.xml verify -Pitest -Dfailsafe.useFile=false | grep -vF "[INFO] Downloading:" | grep
+- mvn -s .travis.maven.settings.xml verify -Pitest -Dfailsafe.useFile=false -Dlicense.skip=true | grep -vF "[INFO] Downloading:" | grep
```

Comment:
I understand the frustration with the license check, but I don't think it's a good idea to skip it. We better suffer a bit more and fix whatever there is to fix there, instead of just ignoring this.